automation-test.py

```diff
@@ -0,0 +1,96 @@
+import asyncio
+
+import time
+
+import secterfunction
+
+import automationsectername
+
+from pyppeteer import launch
+
+async def main():
+    
+    browser = await launch({'headless':False})
+    
+    page = await browser.newPage()
+    
+    await page.setViewport({ "width": 1080, "height": 1024 })
+    
+    await page.goto(automationsectername.facebookurl)
+    
+    await page.waitForSelector(automationsectername.loginemailselecter)
+    
+    await page.type(automationsectername.loginemailselecter,automationsectername.loginemail )
```

Comment:
dont do automationsectername.loginemailselecter instead use only loginemailselecter

you can achieve this by doing the following import

import loginemailselecter from automationsectername